Formula/vowpal-wabbit.rb

```diff
@@ -26,14 +30,18 @@ def install
     mkdir "build" do
       system "cmake", "..", *std_cmake_args,
                             "-DBUILD_TESTS=OFF",
-                            "-DRAPIDJSON_SYS_DEP=ON"
+                            "-DRAPIDJSON_SYS_DEP=ON",
+                            "-DFMT_SYS_DEP=ON",
+                            "-DSPDLOG_SYS_DEP=ON",
+                            "-DBUILD_FLATBUFFERS=ON"
       system "make", "install"
     end
     bin.install Dir["utl/*"]
     rm bin/"active_interactor.py"
     rm bin/"new_version"
     rm bin/"vw-validate.html"
     rm bin/"clang-format"
+    rm_r bin/"flatbuffer"
```

Comment:
I assume we need to remove all these things because of the
```ruby
bin.install Dir["utl/*"]
```
above. Would it be easier if we just specified the things we wanted to install from `utl`, rather than install them all the remove the ones we don't want?

Reply:
It's a good point but there's quite a number more tools in the directory than things that are removed.